torchaudio/__init__.py

```diff
@@ -128,13 +149,17 @@ def save(filepath, src, sample_rate, precision=16, channels_first=True):
         channels_first (bool): Set channels first or length first in result. (
             Default: ``True``)
     """
-    si = sox_signalinfo_t()
-    ch_idx = 0 if channels_first else 1
-    si.rate = sample_rate
-    si.channels = 1 if src.dim() == 1 else src.size(ch_idx)
-    si.length = src.numel()
-    si.precision = precision
-    return save_encinfo(filepath, src, channels_first, si)
+
+    if get_audio_backend() == "sox":
+        func = _sox_backend.save
```

Comment:
You could do
```
getattr(get_audio_backend_module() , 'save')
```

Do we want get_audio_backend()  to return a module or a string that is the name?

A module can yield the name as well via introspection